setup/install/providers/kubernetes-v2/aws-eks.md

```diff
@@ -0,0 +1,200 @@
+---
+layout: single
+title:  "Amazon EKS"
+sidebar:
+  nav: setup
+---
+
+
+
+{% include toc %}
+
+
+
+
+
+> Before you proceed with installing Spinnaker on EKS, we strongly recommend that you familiarize yourself with [Amazon EKS concepts](https://docs.aws.amazon.com/eks/latest/userguide/getting-started.html)
+
+
+
+Below are the instructions to be followed if you want to configure [Kubernetes V2 (manifest based) Clouddriver](/setup/install/providers/kubernetes-v2) 
+Spinnaker to deploy to [Amazon EKS](https://aws.amazon.com/eks/).
+
+These instructions assumes that you have AWS CLI [installed](https://docs.aws.amazon.com/cli/latest/userguide/installing.html) ,
+[configured](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html) and have access to managing and each of the managed account.
+
+## Managing Account - Base Setup
+
+In Managing Account Create a two subnet VPC , IAM Roles, Instance Profiles and Security Group for EKS control plane communication and an EKS Cluster
```

Comment:
Here's an attempt at a rewrite for this sentence:

"In the managing account, create a two-subnet VPC, IAM roles, instance profiles, and A Security Group for EKS control-plane communications and an EKS cluster."

But I'm pretty confused about what goes with what, especially the end "and an EKS cluster." 
Are you creating all these things...
* VPC
* IAM roles
* instance profiles
* Security Group
...for control plane communicatons and for an EKS cluster?